test/e2e/nft/nft.spec.js

```diff
@@ -1,33 +0,0 @@
-const { strict: assert } = require('assert');
```

Comment:
This test was to ensure we was testing a build with the NFT feature enabled, this is no longer needed as this feature is in the main build and no longer behind a feature flag